demisto_sdk/commands/test_content/test_modeling_rule/test_modeling_rule.py

```diff
@@ -1026,16 +1023,22 @@ def validate_modeling_rule(
             missing_event_data, _ = is_test_data_exists_on_server(
                 modeling_rule.testdata_path
             )
-
-            # initialize xsiam client
-            xsiam_client_cfg = XsiamApiClientConfig(
-                base_url=xsiam_url,  # type: ignore[arg-type]
-                api_key=api_key,  # type: ignore[arg-type]
-                auth_id=auth_id,  # type: ignore[arg-type]
-                token=xsiam_token,  # type: ignore[arg-type]
-                collector_token=collector_token,  # type: ignore[arg-type]
-            )
-            xsiam_client = XsiamApiClient(xsiam_client_cfg)
+            modeling_rule_is_compatible = validate_modeling_rule_version_against_tenant(
+                to_version=modeling_rule.to_version,
+                from_version=modeling_rule.from_version,
+                tenant_xsiam_version=tenant_xsiam_version)
+            if not modeling_rule_is_compatible:
+                # Modeling rule version is not compatible with the XSIAM version of the tenant, skipping
+                skipped = (
+                    f'Skipping the Modeling Rule {modeling_rule} due to version mismatch'
+                )
+                logger.warning(f'[yellow]{skipped}[/yellow]', extra={'markup': True})
+                test_case = TestCase('Modeling Rule not compatible with tenant\'s XSIAM version',
```

Comment:
write the from/to versions to the skipped result text

Reply:
I added this [here](https://github.com/demisto/demisto-sdk/pull/3898/files#diff-b2bc81209d5ef043aa657993c81f1e5ed7b620a388230d04a8786311737ea137R214), that way the string representation of the ModelingRule object will always print the from/to version.